ui/tree/css/_tree.scss

```diff
@@ -65,7 +95,7 @@ $c-blunder: #df5353;
     font-style: italic;
   }
 
-  move:not(.empty):hover {
+  move:not(.empty):not(.blunder):not(.mistake):not(.inaccuracy):not(.good):not(.brilliant):not(.interesting):hover {
```

Comment:
is it possible to have the list inside the :not selector ?

```suggestion
  move:not(.empty, .blunder, .mistake, .inaccuracy, .good, .brilliant, .interesting):hover {
```

Reply:
Apparently not ?
![image](https://github.com/user-attachments/assets/b184985d-60e4-4a6b-a597-1882859c1714)

Reply:
It looks like major browsers are supporting it with a version released ~ end of 2020.
If you want to ensure people using Opera Android or old browsers, you can revert.

Reply:
okay, its a minor on hover change, so will just leave as is.